R/install-remote.R

```diff
@@ -32,11 +32,23 @@ install_remotes <- function(remotes, ...) {
 # Add metadata
 add_metadata <- function(pkg_path, meta) {
   path <- file.path(pkg_path, "DESCRIPTION")
-  desc <- read_dcf(path)
+  if (file.exists(path)) {
+    desc <- read_dcf(path)
+
+    desc <- modifyList(desc, meta)
+
+    write_dcf(path, desc)
+  }
 
-  desc <- modifyList(desc, meta)
+  path <- file.path(pkg_path, "Meta", "package.rds")
+  if (file.exists(path)) {
```

Comment:
I actually did this without an else on purpose because I wanted to add the metadata to _both_ the binary description as well as the text description in the same way. Some functions only look at the text DESCRIPTION, others will use the binary if it is available, so I wanted the same results regardless.